kratos/tests/cpp_tests/processes/test_integration_values_extrapolation_to_nodes_process.cpp

```diff
@@ -99,17 +105,15 @@ namespace Kratos
         */
         KRATOS_TEST_CASE_IN_SUITE(TestIntegrationValuesExtrapolationToNodesProcessTriangle, KratosCoreFastSuite)
         {
+            KRATOS_SKIP_TEST_IF_NOT(KratosComponents<Element>::Has("UpdatedLagrangianElement2D3N")) << "This test needs the StructuralMechanicsApplication" << std::endl;
```

Comment:
Since when do we have this?

Reply:
#5929

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/78353667-bcc2e200-75aa-11ea-9e15-fa163ab6141d.png)

Reply:
now we only need a proper test in Cpp if an app is imported

Reply:
I will update the Meshing with the new mechanism

Reply:
See #6681